### DISCUSSION TITLE
Improve documentation on figure labels for cross-referencing

### DISCUSSION THREAD

**Discussion by tklebel** [quarto-dev/quarto-web#506] [Ideas]
Transitioning from RMarkdown, it took me ages to understand that the labels for figures always need to have the prefix `fig-` for the figure numbering to work

This works:
```
#| label: fig-plots
#| fig-cap: "First figure."
plot(1:5)
```
This does not:
```
#| label: test-fig
#| fig-cap: "Second figure."
plot(1:6)
```

I was treating the labels similar to chunk labels in RMarkdown, and although the examples use the prefix throughout, the top paragraph on [the docs](https://quarto.org/docs/authoring/cross-references.html) only mentions a unique label, not a unique label *that is prefixed with `fig-`*:
> Cross-references make it easier for readers to navigate your document by providing numbered references and hyperlinks to various entities like figures and tables. *Every cross-referenceable entity requires __a label (unique identifier)__ and caption (description).*

It is somewhat mentioned in the "note" box below, but imho this is too low down in the text. Maybe the latter sentence could be amended with "requires a label (unique identifier prefixed with `fig-`) and caption..."?

**Comment by jjallaire** [comment]
Thanks for pointing this out! Addressed here: https://github.com/quarto-dev/quarto-web/commit/d39c9770f5223a5587b9c3309a4d0ebf81511d11

**Comment by tklebel** [comment]
Awesome, thanks!
